docs/flip-flop_conversions.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: default
+title: Flip-Flop Conversions
+comments: true
+nav_order: 14
+---
+
+# Flip-Flop Conversions
+{: .no_toc }
+
+This page will teach you how to convert one type of Flip flop to another type.
```

Comment:
No need for capital `Flip flop`, `flip flop` is fine. Note all instances of this.